src/praisonai-agents/praisonaiagents/runtime/registry.py

```
@@ -114,114 +122,266 @@ def clear(self) -> None:
+            
+            with _registry_lock:
+                if ep.name not in _runtimes:
+                    _runtimes[ep.name] = make_loader(ep)
+                    _runtime_metadata[ep.name] = {"entry_point": str(ep)}
+                    
+    except ImportError:
+        # entry_points not available in older Python versions
+        pass
+    except Exception:
+        # Ignore discovery errors - plugins are optional
+        pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)